dspace-spring-rest/src/main/java/org/dspace/app/rest/submit/SubmissionService.java

```diff
@@ -160,4 +183,51 @@ public UploadBitstreamRest buildUploadBitstream(ConfigurationService configurati
         return data;
     }
 
+    public XmlWorkflowItem createWorkflowItem(Context context, Request currentRequest)
+            throws SQLException, AuthorizeException, WorkflowException {
+        Reader reader = null;
+        XmlWorkflowItem wi = null;
+        try {
+            reader = currentRequest.getHttpServletRequest().getReader();
```

Comment:
we discuss that some times ago deciding to unblock other PRs simply allowing to copy and paste this code. Refactor this code in a common utility class or identify a better strategy to automatically bind the request body to the uri-list format is for sure in the wish list but for consistency with the previous decision I would argue that it is out of scope of this PR

Reply:
Could we again capture this as a TODO?  I'm worried these tasks will either be forgotten or we'll overlook areas that need updating.  I think we either should create a quick utility method (just copy this exact code elsewhere) *now*, or somehow make this section of code findable so we can refactor it in the future.

Reply:
Also, I've just remembered that a PR for Configurable Entities (#2331) creates a parsing method for `text/uri-list` and adds a basic IT. So, this will end up getting solved eventually....but, we should leave a TODO or something to ensure this area of code gets cleaned up.

If you want to see how PR#2331 handles this parsing, it's in the `Utils` class here: https://github.com/DSpace/DSpace/pull/2331/files#diff-6945a4795213cbdd5cf14c8f50625ee9R241
It also has an integration test in that PR in this commit: https://github.com/DSpace/DSpace/pull/2331/commits/65a94cdda7ed64b92d9d2c15dff224e8cff9c4b9

So, for the purposes of this PR, I'd recommend we just add a TODO, and we can fix this later once all these various open PRs get merged into `master`.

Reply:
done